docs/docsite/rst/plugins/action.rst

```diff
@@ -1,16 +1,15 @@
 .. _action_plugins:
 
-Action Plugins
+Action plugins
 ==============
 
 .. contents::
    :local:
    :depth: 2
 
-Action plugins act in conjunction with :ref:`modules <working_with_modules>` to execute the actions required by playbook tasks.
-They usually execute automatically in the background doing prerequisite work before modules execute.
+Action plugins act in conjunction with :ref:`modules <working_with_modules>` to execute the actions required by playbook tasks. They usually execute automatically in the background doing prerequisite work before modules execute.
```

Comment:
Maybe we should add something along the lines of "action plugin is the control node part of a module"? This should help explain why action plugins share their documentation with the module.